generators/workspace/templates/plain_initialize_once/deploy/web/404.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en" style="height: 100%;">
+<head>
+    <meta charset="UTF-8">
+    <title>Error 404</title>
+    <link href="https://fonts.googleapis.com/css?family=Yantramanav:700,400" rel="stylesheet" type="text/css">
+    <style>
+      #error-box {
+        position: absolute;
+        text-align: center;
+        width: 40%;
+        background-color: white;
+        color: #252759;
+        border-radius: 5px;
+        padding: 20px;
+      }
+
+      #error-box::after {
+        position: absolute;
+        top: 0;
+        left: 0;
+        content: '';
+        clip-path: polygon(75% 0, 100% 0, 100% 100%, 25% 100%);
+        box-shadow: inset 0px 0px 0px 2px currentColor;
+        color: #FFD743;
+        border-radius: 5px;
+        height: 100%;
+        width: 100%;
+      }
+
+      #error-box::before {
+        position: absolute;
+        top: 0;
+        left: 0;
+        content: '';
+        clip-path: polygon(0 0, 75% 0, 35% 100%, 0 100%);
+        box-shadow: inset 0px 0px 0px 2px currentColor;
+        color: #F24405;
+        border-radius: 5px;
+        width: 100%;
+        height: 100%;
+      }
+    </style>
+</head>
+<body style="height: 100%; font-family: 'Yantramanav', 'sans-serif';">
+<div class="container" style="height:100%">
+  <main style="height: 100%">
+    <div style="height: 100%; display: flex; align-items: center; justify-content: center;">
```

Comment:
Maybe we can move the content of the `style` attributes to the `<style>` section?